src/utils/FormInputHandling.ts

```diff
@@ -15,16 +22,108 @@ async function getData(currentPage: number, limit: number) {
   };
 }
 
+// getting data from db when session_id is null
+async function getDataNoLinks(currentPage: number, limit: number) {
```

Comment:
ideally it should be `getDataWithoutIds` .. the current name is confusing.

Reply:
renamed it